.github/workflows/wheels-test.yaml

```
@@ -69,6 +69,8 @@ jobs:
               - { ARCH: 'amd64', PY_VER: '3.9', CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu18.04', gpu: 'v100', driver: 'latest' }
               - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '12.0.1', LINUX_VER: 'ubuntu20.04', gpu: 'v100', driver: 'latest' }
               - { ARCH: 'arm64', PY_VER: '3.9', CUDA_VER: '12.0.1', LINUX_VER: 'ubuntu20.04', gpu: 'a100', driver: 'latest' }
+              - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu20.04', gpu: 'v100', driver: 'latest' }
+              - { ARCH: 'arm64', PY_VER: '3.9', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu20.04', gpu: 'a100', driver: 'latest' }
```

Since the other arm64 job for CUDA 12.0.1 is testing Python 3.9, it would be ideal to test Python 3.10 here (spread the coverage across the matrix).
```suggestion
              - { ARCH: 'arm64', PY_VER: '3.10', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu20.04', gpu: 'a100', driver: 'latest' }
```

However, it's important to note that our arm64 resources are limited and we have an exception to the policy I mentioned above. We don't have any PR tests for CUDA 11 on ARM, so that implies that we have a goal of running only one ARM job for wheels per pull request. If so, we could only test CUDA 12.2 and skip testing 12.0 (so the number of ARM test jobs stays constant). And if it's true that we only want one ARM CI job for wheels in pull requests, then I would prefer to only test the latest supported Python version (currently 3.10).

---

@jameslamb What are your thoughts here? Keep this to one ARM job? If so, should we test only Python 3.10?

---

blegh I made a mistake here. I agree with you it should be:

* only 1 `arm` job on PRs
* that job should use CUDA 12.2 + Python 3.10

I meant to do that, actually... that job being Python 3.9 here was a mistake. Just pushed https://github.com/rapidsai/shared-workflows/pull/166/commits/b3911076b7c2cf88729389495fd1831a20099eaf fixing that.